packages/@lwc/engine/src/shared/format.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { isTrue, isNull, ArrayJoin, ArrayPush, StringToLowerCase } from '@lwc/shared';
+
+import { VM, UninitializedVM } from '../framework/vm';
+
+export function getComponentTag(vm: UninitializedVM, withIndex = false): string {
```

Comment:
I am tempted to remove the second parameter `withIndex`. 

It is currently, needed to keep the backward compatibility with the performance marks the engine produces in `DEV` mode. Now that I am thinking about it, the VM `idx` is an internal property that doesn't bring any value to the mark messages. Since those messages are only produced in DEV, the risk of breaking a consumer is almost null. What do you think about dropping the `idx` from the `DEV` marks?

Reply:
go for it! the boolean argument was never an option for me anyways :)